.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "rules": {
+    "@next/next/no-img-element": "off"
```

Comment:
We had warnings in a few places, but since the image optimisation is disabled I guess we can disable this rule